pkg/qe-tests/cypress/integration/tests/Rhv/edit_mapping_test.ts

```diff
@@ -0,0 +1,31 @@
+import { MappingNetwork } from '../../models/mappingNetwork';
+import { editNetworkMapping, editStorageMapping, testData } from './config_separate_mapping_rhv';
+import { login } from '../../../utils/utils';
+import { MappingStorage } from '../../models/mappingStorage';
+
+describe('Creating CRUD operation', () => {
```

Comment:
CRUD operations include creating .
So just "CRUD Operations" is fine.



      describe('CRUD operations', () => {

Reply:
Thanks a lot, @sshveta for the suggestion. I've amended those changes and pushed.
Would you please review it again? Thanks in advance !